tests/test_default_qubit.py

```diff
@@ -847,7 +847,7 @@ def test_supported_observables(self):
             dev.reset()
 
             op = getattr(qml.ops, g)
-            if op.num_wires == 0:
+            if op.num_wires <= 0:
                 wires = [0]
             else:
                 wires = list(range(op.num_wires))
```

Comment:
In `test_default_qubit.py`, the following test:

```python
    def test_unsupported_gates(self):
         self.logTestName()
         dev = qml.device("default.qubit", wires=2)
         gates = set(dev._operation_map.keys())
         all_gates = set(qml.ops.__all_ops__)
         for g in all_gates - gates:
            op = getattr(qml.ops, g)

            if op.num_wires == 0:
                wires = [0]
            else:
                wires = list(range(op.num_wires))
```
Should we not change the wires check to: 

```python
            if op.num_wires < 0:
                wires = [0]
            elseif op.num_wires == 0:
                wires = list(range(op.num_wires))
```

Reply:
the same comment for `test_unsupported_observables(self)` test function

Reply:
Yep you're right :) Our thinking is that it's not worth fixing this test due to the upcoming test refactor (it will be replaced outright)

Reply:
@josh146 , there is still inconsistency in the test, even if we are going to refactor it later. We can not use `wires=[0]` for both `<0` and `=0` condition as `All=0`...we will not be assigning wires properly. Till we refactor, we still have to use correct tests, no?

Reply:
Good call, I've made the change :+1: